### DISCUSSION TITLE
Use plugins like daisyUI with Tailwind Standalone version?

### DISCUSSION THREAD

**Discussion by ddahan** [tailwindlabs/tailwindcss#8896] [General]
I'm trying to use [Tailwind Standalone version](https://tailwindcss.com/blog/standalone-cli) with plugins like [daisyUI](https://daisyui.com/) (for many reasons I don't want to use their CDN version).

At this point, I may be wrong, but I think there is no other way than building a standalone [Tailwind+DaisyUI] package by myself, using the same [pkg util](https://github.com/vercel/pkg) that Tailwind team uses itself. Am I wrong?

I'd like to ask to the Tailwind team: before trying the journey by myself, is there any instructions I should be aware of to build my own Tailwind with pkg? Any configuration? Gotchas?

Thanks :)

**Comment by pejeio** [comment]
Hi @ddahan! Did you find a solution?

**Reply by ddahan** [reply]
Hi! It seems [someone else had](https://github.com/saadeghi/daisyui/issues/770#issuecomment-1897624457).
